.github/workflows/main.yml

```diff
@@ -27,5 +27,15 @@ jobs:
       - name: Docker 
         run: docker build -t verify-trusty . --file docker/seahorn-dev10-verify-trusty.Dockerfile
       
-      - name: Run Tests 
-        run: docker run -t verify-trusty /bin/bash -c "cd build && mkdir -p /tmp/verifyTrusty && env VERIFY_FLAGS=\"${{ matrix.verify_flag }}\" ctest --output-on-failure --timeout 2000 ."
+      - name: Get number of Cores
+        id: cores
+        run: echo "::set-output name=num_cores::$(nproc --all)"
+      
+      - if: ${{ matrix.verify_flag == '--cex' }}
```

Comment:
no. this only applies in CI. We specifically have local sea.yaml files to change options per individual proof. Please use these.